internal/pgbouncer/reconcile.go

```diff
@@ -146,10 +149,15 @@ func Pod(
 		),
 	}
 
+	mkdirCommand := ""
+	if logfile != "" && filepath.Dir(logfile) != "/tmp" {
+		mkdirCommand = shell.MakeDirectories("/tmp", filepath.Dir(logfile)) + "; "
+	}
```

Comment:
This is confusing me... If the provided `logfile` doesn't have `/tmp` as the base, why would we want to force it to use `/tmp` as the base? :thinking: